pkg/api/client/activities/get_activities_responses.go

```diff
@@ -30,7 +30,7 @@ func (o *GetActivitiesReader) ReadResponse(response runtime.ClientResponse, cons
 		}
 		return result, nil
 	default:
-		return nil, runtime.NewAPIError("response status code does not match any response statuses defined for this endpoint in the swagger spec", response, response.Code())
+		return nil, runtime.NewAPIError("[POST /public/v1/query/list_activities] GetActivities", response, response.Code())
```

Comment:
This seems odd. Maybe this should be templated as "Error while calling [POST ...etc..."? My main issue with this is it doesn't read like an error message. But maybe it's okay?

Reply:
I didn't actually change anything here... so I assume it's because we're using different version of `go-swagger`. We should consider building in a container so we can pin the version.

```
swagger version
version: v0.30.5
commit: ab6a7885723430004f1d7db6395369b6e7f3370b
```

Reply:
We have instructions for building with `nix`. That's a good way to keep our dependencies pinned: https://github.com/tkhq/go-sdk#with-nix

(but yes, docker can work for this too!)